.github/workflows/openhands-resolver.yml

```diff
@@ -112,7 +112,12 @@ jobs:
 
       - name: Install OpenHands
         run: |
-          python -m pip install --upgrade -r requirements.txt
+          if [[ "${{ github.event.label.name }}" == "review-pr" ]]; then
```

Comment:
@openhands-agent-exp  The only trigger of this step is 'review-pr' label, from what I see. Verify this. If so, you probably don't need the conditional here when installing, just install from github.